docker/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN python get-pip.py
 RUN pip3 install wheel
 
 # Install Pytorch
-RUN pip3 install torch==2.0.0.dev20230103+cu117 torchvision==0.15.0.dev20230103+cu117 --extra-index-url https://download.pytorch.org/whl/nightly/cu117
+RUN pip3 install torch==2.0.0.dev20230219+cu117 torchvision==0.15.0.dev20230219+cu117 --extra-index-url https://download.pytorch.org/whl/nightly/cu117
```

Comment:
Can we just reference the `requirements.txt` so this doesnt need to be manually updated?

Reply:
Used reference to `requirements.txt` to avoid explicit version naming here.